sympy/plotting/pygletplot/color_scheme.py

```diff
@@ -164,12 +165,12 @@ def _fill_in_vars(self, args):
         if len(args) == 0:
             return defaults
         if not isinstance(args, (tuple, list)):
-            raise v_error
+            raise ValueError
```

Comment:
PyCharm doesn't recognize `v_error`, I don't see what it could mean either. It seems that here should be `ValueError` raised, it's implied both by the context and the name.